docs/dataset_curation_guidelines.md

```diff
@@ -0,0 +1,141 @@
+     documentation and display.
+   - **Example**: "Single cell transcriptional and chromatin accessibility profiling redefine 
+     cellular heterogeneity in the adult human kidney"
+   
+3. **CxG Link:**
+   - **Description**: The CellxGene link to access the dataset.
+   - **Example**: "https://cellxgene.cziscience.com/e/13a027de-ea3e-432b-9a5e-6bc7048498fc.cxg/"
+   
+4. **h5ad link:**
+   - **Description**: The direct link to the `.h5ad` data file of the dataset.
+   - **Example**: "https://datasets.cellxgene.cziscience.com/dabd979f-cc50-4526-81f3-8bc6c673ca36.h5ad"
+   
+5. **Reference_DOI:**
+   - **Description**: The DOI reference for the associated publication(s) for the dataset.
+   - **Example**: "DOI: 10.1038/s41467-021-22368-w"
+   
+6. **Bionetworks reference:**
+   - **Description**: Indicate whether the dataset has a reference within the Bionetworks repository.
+   - **Example**: "T" (True)
+   
+7. **Standard category present? (T/F):**
+   - **Description**: Flag indicating whether standard categories are present in the dataset.
+   - **Example**: "T" (True)
+   
+8. **Standard category cell_type present? (T/F):**
+   - **Description**: Flag indicating whether the standard category for cell type is present in 
+     the dataset.
+   - **Example**: "T" (True)
+   
+9. **Author Category Cell Type Field Name:**
+   - **Description**: This column shows the name of the field as it appears in the Dataset 
+     Explorer UI. It indicates which specific field within the dataset corresponds to a certain 
+     category, such as "cell type" or other annotations. Fields marked as `Cell types` in the 
+     `Content` column play a key role in graph generation using the `pandasaurus_cxg` library, which 
+     is employed in the data pipeline.
+   - **Example**: "author_cell_type"
+   
+10. **Content:**
+    - **Description**: This column indicates whether the field is used for cell type annotations 
+      or for other dataset annotations (e.g., Cell type or Other).
+    - **Example**: "Cell types"
+    
+11. **Value type(s):**
+    - **Description**: This column specifies if the values in the dataset are represented in full 
+      names or as abbreviations.
+    - **Example**: "abbreviations"
+    
+12. **Notes:**
+    - **Description**: Any additional notes or comments regarding the dataset.
+    - **Example**: "Muto et al. (2021) Nat Commun"
+    
+13. **Study Short Name:**
+    - **Description**: The shortened name or acronym of the study associated with the dataset.
+    - **Example**: "Muto et al. (2021) Nat Commun"
+    
+14. **CxG Dataset Collection X:**
+    - **Description**: The CellxGene link to the collection where the dataset is stored.
+    - **Example**: "https://cellxgene.cziscience.com/collections/9b02383a-9358-4f0f-9795-a891ec523bcc"
+    
+15. **Is the dataset Normal or Normal/Diseased:**
+    - **Description**: Indicates whether the dataset is of normal samples or includes both 
```

Comment:
Suggest amend: "Indicates whether the dataset includes normal samples, diseased samples or both"